common/fuse.h

```diff
@@ -290,6 +290,13 @@ class Fusor : public Looper {
   Vec3 speed() { return speed_; }  // m/s
 #endif
 
+  float rss() {
```

Comment:
Needs a comment to explain what it does.
(Or a MUCH better name....)

Reply:
renamed to swing_accel() and added comment